tests/with_span_class_option_none.py

```diff
@@ -0,0 +1,17 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from unittest import main, TestCase
+
+
+class Test(TestCase):
+    def test_mdx_span_class_none_option(self):
+        from markdown import markdown
+        from mdx_unimoji import UnimojiExtension
```

Comment:
When debugging I suspected `markdown` was doing something janky and trying to cache extensions or something. I'll move them.